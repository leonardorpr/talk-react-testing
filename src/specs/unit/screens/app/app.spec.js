import React from 'react'
import { render } from 'react-testing-library'

import App from 'screens/app'

describe('[SCREENS] App', () => {
  it('Should render correctly', () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })
})
