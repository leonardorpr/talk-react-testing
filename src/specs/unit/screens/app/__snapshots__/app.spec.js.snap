// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[SCREENS] App Should render correctly 1`] = `
<div>
  Hello World
</div>
`;
